File: Remi_Hasan/src/main/java/model/stylesheet/Page.java
package model.stylesheet;

import java.util.List;

public class Page {

    public final String identifier;
    private final List<Default> defaults;
    private final List<Section> sections;

    public Page(String identifier, List<Default> defaults, List<Section> sections) {
        this.identifier = identifier;
        this.defaults = defaults;
        this.sections = sections;
    }
}

File: niro/src/main/scala/nl/uva/se/sc/niro/model/qls/Page.scala
package nl.uva.se.sc.niro.model.qls

import nl.uva.se.sc.niro.model.ql.AnswerType

case class Page(name: String, sections: Seq[Section], defaultStyles: Map[AnswerType, Styling]) {

  def collectAllQuestions(): Seq[Question] = {
    sections.flatMap(_.collectAllQuestions)
  }
}
